Loadouts/Blufor/adf_auga3_auscam.hpp

```diff
@@ -139,7 +153,7 @@ class ftl: rifleman {// FTL
   backpack[] = {CAMO_BACKPACK_FTL};
   weapons[] = {GLRIFLE};
   magazines[] = {GLRIFLE_MAG,GLRIFLE_MAG_HE,GLRIFLE_MAG_SMOKE,LEADER_GRENADES};
-  items[] += {LEADER_TOOLS};
+  items[] += {LEADER_TOOLS,"acex_intelitems_notepad"};
```

Comment:
should we just add notepad to LEADER_TOOLS
seems harmless enough, it's practically zero weight anyway

Reply:
I think I'm the only one who actually puts the notepad in my loadouts lmao. I wouldn't be against it, I can do a quick update to add it to common.hpp if we want.

Reply:
yea, and I like when i have it, I think i'd pref to make it standardized because then we have consistency in kits, and with it being zero weight/not a balance concern I say we should standardize on it

Reply:
done